src/core/dom/fake/element.js

```diff
@@ -1,19 +1,21 @@
 import {FakeClassList} from './class.list';
 
+const emptyRect = {
+	left: 0,
+	right: 0,
+	top: 0,
+	bottom: 0,
+	width: 0,
+	height: 0
+};
+
 export class FakeElement {
 	constructor() {
 		this.classList = new FakeClassList();
 	}
 
 	getBoundingClientRect() {
-		return {
-			left: 0,
-			right: 0,
-			top: 0,
-			bottom: 0,
-			width: 0,
-			height: 0
-		};
+		return emptyRect;
```

Comment:
Is there a case when bounding rect is modified from outside?
Maybe emptyRect props should be frozen?